### DISCUSSION TITLE
Support policy for System.* nuget libraries

### DISCUSSION THREAD

**Discussion by Tobias-08** [dotnet/runtime#68900] [Q&A]
Hi,

I am wondering what the support policy for System.* libraries is that are available via nuget.

There are libraries that have a version that corresponds to current .NET (6), e. g. https://www.nuget.org/packages/System.Diagnostics.DiagnosticSource/6.0.0 . But there are other libraries that do not have a corresponding version, e. g. https://www.nuget.org/packages/System.ComponentModel.Annotations/5.0.0 .

In the latter case: What version am I supposed to use with .NET 6 (ASP.NET Core 6 in my case)? The 5 version might imply that its support might end at the same time as for .NET 5?

Thanks!
Tobias

**Comment by huoyaoyuan** [comment]
> In the latter case: What version am I supposed to use with .NET 6 (ASP.NET Core 6 in my case)? The 5 version might imply that its support might end at the same time as for .NET 5?

For `System.CommonentModel.Annotations`, and also `Microsoft.Win32.Registry` and so on, there are shipped in box of .NET 6, and was already there several versions ago. The nuget versions were used for supporting older frameworks.
